index.html

```diff
@@ -3682,7 +3682,7 @@ <h3>Ecosystem Compatibility</h3>
 AnonCreds</a>,
 <a href="https://www.ietf.org/archive/id/draft-mcnally-envelope-02.html">
 Gordian Envelopes</a>, and
-<a href="https://www.ietf.org/archive/id/draft-ssmith-acdc-02.html">
+<a href="https://www.ietf.org/archive/id/draft-ssmith-acdc-03.html">
```

Comment:
internet drafts are not stable identifiers - there's no guarantee they'll exist at some point in the future. is there a more stable identifier we could refer to? Perhaps https://datatracker.ietf.org/doc/draft-ssmith-acdc/